docker-compose.yml

```diff
@@ -9,14 +9,8 @@ services:
     cap_add:
       - NET_ADMIN
     network_mode: 'host'
-    environment:
-      LOGGER_LEVEL: debug
-      PPROF_ENABLED: "true"
-      HOSTNAME: link-1
-      ETCD_HOSTS: http://172.17.0.1:32379
-      INTERFACE: eth10
-      USER: link
-      PASSWORD: password
+    env_file:
+      - .env
```

Comment:
Oh yeah not sure i should have commited that. I used an envfile so that docker-compose and VSCode could import those variable. This let me run link-1 in debug mode from vscode.